llvm/lib/Target/AArch64/AArch64ISelLowering.cpp

```diff
@@ -29515,6 +29509,30 @@ AArch64TargetLowering::LowerPARTIAL_REDUCE_MLA(SDValue Op,
   return DAG.getNode(ISD::ADD, DL, ResultVT, Acc, Extended);
 }
 
+SDValue
+AArch64TargetLowering::LowerGET_ACTIVE_LANE_MASK(SDValue Op,
+                                                 SelectionDAG &DAG) const {
+  EVT VT = Op.getValueType();
+  assert(VT.isFixedLengthVector() && "Expected fixed length vector type!");
+
+  auto &Subtarget = DAG.getSubtarget<AArch64Subtarget>();
```

Comment:
This will generate an "unsused variable" compiler warning for release builds.  It is best to have everything in the assert and in this case I think `AArch64TargetLowering` already has a `Subtarget` member you can use?